src/plugins/intel_gpu/src/kernel_selector/core/actual_kernels/reorg_yolo/reorg_yolo_kernel_ref.cpp

```diff
@@ -38,23 +38,20 @@ ReorgYoloKernelRef::DispatchData SetDefault(const reorg_yolo_params& params) {
     std::vector<std::vector<Tensor::DataChannelName>> dims_by_gws;
 
     const auto& input = params.inputs[0];
-    if (input.GetLayout() == DataLayout::bfyx) {
-        dispatchData.gws = {input.X().v, input.Y().v, input.Feature().v};
-        dims_by_gws = {{Tensor::DataChannelName::X},
-                       {Tensor::DataChannelName::Y},
-                       {Tensor::DataChannelName::FEATURE}};
-    } else {
-        dispatchData.gws = {input.Feature().v * input.Batch().v, input.X().v, input.Y().v};
-        dims_by_gws = {{Tensor::DataChannelName::FEATURE, Tensor::DataChannelName::BATCH},
-                       {Tensor::DataChannelName::X},
-                       {Tensor::DataChannelName::Y}};
-    }
+    dispatchData.gws = {input.X().v, input.Y().v, input.Feature().v};
+    dims_by_gws = {{Tensor::DataChannelName::X},
+                   {Tensor::DataChannelName::Y},
+                   {Tensor::DataChannelName::FEATURE}};
```

Comment:
why the other condition removed?

Reply:
That condition was used for yxfb branch in the kernel. When I added unit tests, it turned out that yxfb branch (see [line 44 and below in old kernel](https://github.com/openvinotoolkit/openvino/pull/12463/files#diff-ca077a806df800ff766d983a6d4990f3d36c130b3d23f4968d72029750a9b74dL44)) works incorrect. I removed that buggy code and handled yxfb layout using _GET_INDEX() macro. So now that condition is not needed anymore.

Reply:
But it will affect the performance because original code is processing F*B first and then X and then Y which is aligned with the layout. If it accesses X, => Y than the perf will get bad. Could you revert the original conditions?

Reply:
@yeonbok  Are you sure that it was used? As I understood logic was presented but worked incorrectly

Reply:
"Worked incorrectly" <= means that on the master branch before your modification added?

Reply:
Exactly. You can add reorg_yolo_gpu_test.cpp from this PR to master branch (currently there is no unit tests for reorg_yolo in master) and run yxfb test to check.